src/components/BlogList.vue

```diff
@@ -65,24 +31,23 @@ onMounted(() => {
   fetchArticles();
 });
 
-watch(
-  () => props.searchQuery,
-  (newQuery) => {
-    page.value = 1;
-    fetchArticles();
-  }
-);
+function formatDate(timestamp: number) {
+  const date = new Date(timestamp * 1000);
+  return new Intl.DateTimeFormat('en-US', {
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+  }).format(date);
+}
 </script>
 
-
-
 <template>
   <div class="blog-list">
     <div 
-    v-for="(article, index) in articles" 
-    :key="index" 
-    class="blog-item"
-    @click="navigateToBlog(article.id)" 
+      v-for="(article, index) in articles" 
+      :key="index" 
```

Comment:
We should always avoid `index` as keys in template loops. See [why](https://stackoverflow.com/questions/44531510/why-not-always-use-the-index-as-the-key-in-a-vue-js-for-loop#:~:text=Because%20arrays%20are%20mutable.,better%20than%20using%20an%20index.)

What you can do is you can use `uuidv4` library to generate unique key for each articles and then use the id as keys